.devcontainer/Dockerfile

```diff
@@ -29,6 +29,12 @@ RUN wget https://dl.min.io/server/minio/release/linux-$([ "$(uname -m)" = "aarch
     && chmod +x minio \
     && mv minio /usr/local/bin/minio
 
+RUN curl -fsSL -o fake-gcs-server_1.52.2.tar.gz https://github.com/fsouza/fake-gcs-server/releases/download/v1.52.2/fake-gcs-server_1.52.2_Linux_$([ "$(uname -m)" = "aarch64" ] && echo "arm64" || echo "amd64").tar.gz \
+    && mkdir -p fake-gcs-server \
+    && tar zxf fake-gcs-server_1.52.2.tar.gz -C fake-gcs-server \
+    && chmod +x fake-gcs-server/fake-gcs-server \
+    && mv fake-gcs-server/fake-gcs-server /usr/local/bin/fake-gcs-server
```

Comment:
I think it would be good to rm the `fake-gcs-server` dir you have created?

Reply:
Good suggestion. Updated.